go.sum

```diff
@@ -20,20 +20,15 @@ github.com/DataDog/datadog-go v2.2.0+incompatible/go.mod h1:LButxg5PwREeZtORoXG3
 github.com/Jeffail/gabs v1.1.1 h1:V0uzR08Hj22EX8+8QMhyI9sX2hwRu+/RJhJUmnwda/E=
```

Comment:
Did you do a go mod tidy? :D

Reply:
Probably, the `ci/prom/vendor` job should catch anything inconsistent.